app/mm-redux/reducers/entities/groups.ts

```diff
@@ -196,7 +196,22 @@ function groups(state: Dictionary<Group> = {}, action: GenericAction) {
         }
         return nextState;
     }
+    case GroupTypes.RECEIVED_ALL_GROUPS_ASSOCIATED_TO_CHANNELS_IN_TEAM: {
+        const nextState = {...state};
+        const {groupsByChannelId} = action.data;
+
+        for (const channelID of Object.keys(groupsByChannelId)) {
+            if (groupsByChannelId[channelID]) {
+                for (const group of groupsByChannelId[channelID]) {
+                    nextState[group.id] = group;
+                }
+            }
+        }
```

Comment:
Thanks for the suggestion. I addressed this in the first PR since it's not a change that happened regarding this PR and a change that happened in the first PR. I initially was getting this error:
![image](https://user-images.githubusercontent.com/17804942/81306164-2b350d00-904d-11ea-8e53-6be78db83ff1.png)
so I had to add an `as Group` to get it to work.

This is the commit that addressed this comment https://github.com/mattermost/mattermost-mobile/pull/4118/commits/00991e36ebb597d851c8e37496cb6b56bfa146fb

Reply:
I've rebased this PR on top of the first one so that this fix also shows up in this PR.